sklearn/datasets/_lfw.py

```diff
@@ -231,6 +231,18 @@ def _fetch_lfw_people(
     return faces, target, target_names
 
 
+@validate_params(
+    {
+        "data_home": [str, None],
+        "funneled": ["boolean"],
+        "resize": [float, None],
+        "min_faces_per_person": [int, None],
```

Comment:
Not all ints are valid. You can pass an Interval to specify a range of valid values
```suggestion
        "min_faces_per_person": [Interval(Integral, 0, None, closed="left"), None],
```
Note that you'll need to import Interval from _param_validation and Integral from numbers